### ISSUE TITLE
[Intervention] Ignored attempt to cancel a touchmove event with cancelable=false, for example because scrolling is in progress and cannot be interrupted.

### ISSUE THREAD

**Issue by svooterInfo** [idiotWu/smooth-scrollbar#200]
Thanks for the plugin,

 I ran into an unpleasant bug, can you help with it?

**Steps to reproduce:**
- scroll on any touch device in chrome (mb & others browsers, didn't check)

**Actual result:**
- warning in console (touch.js?a801:30 [Intervention] Ignored attempt to cancel a touchmove event with cancelable=false, for example because scrolling is in progress and cannot be interrupted.)

**Expected result:**
- all is fine & console is clean

The issue is reproducible on allmost any scroll plugin, so sorry if this is dublicate,
Can you add https://github.com/idiotWu/smooth-scrollbar/blob/develop/src/events/touch.ts#L45 some additional check here? e.g 
```
if (willScroll && evt.cancelable) {
  evt.preventDefault; 
  activeScrollbar = scrollbar;
}
```

**Comment by marcushofer** [comment]
Having the same problem. Did you find a solution.

**Comment by idiotWu** [comment]
Fixed in #276
